### ISSUE TITLE
netlifyctl deploy hangs during object upload

### ISSUE THREAD

**Issue by malcolmr** [netlify/netlifyctl#70]
After #67, netlifyctl deploy no longer crashes (thanks!), but it does now hang, even if I run it with a TTY. (In contrast, 0.2.0 worked fine with a TTY, just crashing when run headless).

The output I see is just the following:

```
Counting objects: 352 total objects  ✔
Resolving deltas: 2 objects to upload  ✔
Uploading objects: 0/2 🌍
```

(and then it appears not to make any forward progress, even though the objects it's uploading are tiny: the largest is ~50KB.)

Any idea how to debug this? I've tried running with `--debug`, but that doesn't help because the upload attempt never seems to complete, and so the debug log is never written.

Given the delta between 0.2.0 and 0.2.1, I guess d0ea834 must be the cause, though I've no idea why.

**Comment by malcolmr** [comment]
I note that 0.2.1 _is_ able to do a no-op deploy, which I guess makes sense, just not one that involves uploading any new objects.

**Comment by calavera** [comment]
I'm going to investigate right away. Thanks for reporting it.

**Comment by calavera** [comment]
Sorry for the trouble @malcolmr. I just released 0.2.2 that fixes this issue:

```
➜  web-test2 git:(test) ✗ ../../netlify/netlifyctl/builds/linux-0.2.2/netlifyctl deploy
Counting objects: 2 total objects  ✔
Resolving deltas: 1 objects to upload  ✔
Uploading objects: 1/1 done  ✔
Deploy done  🌎
    https://happy-borg-f26e76.netlify.com
```

**Comment by malcolmr** [comment]
Works fine now, thanks for all the fixes!
